google/api_version_converters.go

```diff
@@ -0,0 +1,80 @@
+package google
+
+import (
+	"encoding/json"
+	"google.golang.org/api/compute/v1"
+)
+
+func convert(item, out interface{}) error {
+	bytes, err := json.Marshal(item)
+	if err != nil {
+		return err
+	}
+
+	err = json.Unmarshal(bytes, out)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func convertInstanceGroupManagerToV1(self interface{}) (*compute.InstanceGroupManager, error) {
```

Comment:
We're going to end up needing a whole bunch of these. Do you think it's worth it to have a separate function for each one instead of inlining it with the code? The majority of this is just error handling, which needs to be done by the caller anyway.